--- unpatched-files/CMakeLists.txt	2025-05-02 15:53:29.171567600 +0200
+++ patched-files/CMakeLists.txt	2025-03-12 15:14:07.677444400 +0100
@@ -1,3 +1,6 @@
+set(CMAKE_C_COMPILER "/app/aflnet/afl-clang-fast")
+set(CMAKE_CXX_COMPILER "/app/aflnet/afl-clang-fast++")
+
 # Copyright 2019 Proyectos y Sistemas de Mantenimiento SL (eProsima).
 #
 # Licensed under the Apache License, Version 2.0 (the "License");
@@ -117,7 +120,7 @@
 ###############################################################################
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_CURRENT_SOURCE_DIR}/cmake/modules)
 if(NOT UAGENT_SUPERBUILD)
-    project(microxrcedds_agent VERSION "3.0.1" LANGUAGES C CXX)
+    project(microxrcedds_agent VERSION "3.0.0" LANGUAGES C CXX)
 else()
     project(uagent_superbuild NONE)
     include(${PROJECT_SOURCE_DIR}/cmake/SuperBuild.cmake)
